### ISSUE TITLE
By default powershell uses TLS 1.0 the site security requires TLS 1.2

### ISSUE THREAD

**Issue by laboratoriodosfundos** [shreyashsaitwal/rush-cli#24]
Sometimes this is necessary to work:
[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12 iwr https://raw.githubusercontent.com/shreyashsaitwal/rush-cli/main/scripts/install/install.ps1 -useb | iex

Here in Windows 7 even with everything updated I had to do it that way.
Here's the tip to want to go through this same difficulty.

**Comment by shreyashsaitwal** [comment]
> [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12

Strange. You shouldn't need to add that manually. TLS 1.2 is enabled by default in the install script:

https://github.com/shreyashsaitwal/rush-cli/blob/3a8d21b87284919a6399412780f315d558bd702e/scripts/install/install.ps1#L20

---

Did you get any errors when you tried to run the install script without explicitly defining the security protocol? If yes, I'd be interested in knowing what the error was.

**Comment by laboratoriodosfundos** [comment]
> > [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
> 
> Strange. You shouldn't need to add that manually. TLS 1.2 is enabled by default in the install script:
> 
> https://github.com/shreyashsaitwal/rush-cli/blob/3a8d21b87284919a6399412780f315d558bd702e/scripts/install/install.ps1#L20
> 
> Did you get any errors when you tried to run the install script without explicitly defining the security protocol? If yes, I'd be interested in knowing what the error was.

Yes, I had problems running as indicated in the tutorial, I spent more than 1 hour trying to solve it and I found out that this was it:
![image](https://user-images.githubusercontent.com/77299764/124605794-ab6a2480-de42-11eb-9cdc-1350dae577f2.png)

You should do some testing of the tool with Windows 7, during the construction of the project the color codes are significantly hindering the use:
![image](https://user-images.githubusercontent.com/77299764/124606372-364b1f00-de43-11eb-9b21-4fb86f2cbc79.png)

**Comment by shreyashsaitwal** [comment]
> Yes, I had problems running as indicated in the tutorial, I spent more than 1 hour trying to solve it and I found out that this was it

Oh, thanks. I will update the installation wiki to include this.

> the project the color codes are significantly hindering the use:

I doubt there's much I can do here, as it's more of an issue with the terminal and not Rush. As a temporary solution, you can try switching to Git Bash. In the future, I might add a flag to disable the console colors.
